services/web/server/src/simcore_service_webserver/projects/_nodes_handlers.py

```diff
@@ -172,6 +172,7 @@ async def create_node(request: web.Request) -> web.Response:
 @login_required
 @permission_required("project.node.read")
 @_handle_project_nodes_exceptions
+# NOTE: Careful, this endpoint is actually "get_node_state," and it doesn't return a Node resource.
```

Comment:
not sure I fully understand the note here.

Reply:
`@routes.get(f"/{VTAG}/projects/{{project_id}}/nodes/{{node_id}}", name="get_node")` on this endpoint I would expect to receive the node resource with all node fields, but this is more like /projects/<project-id>/nodes/<node-id>/state endpoint, where you get just a state of a node.